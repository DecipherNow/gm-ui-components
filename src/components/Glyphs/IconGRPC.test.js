import React from "react";
import { shallow } from "enzyme";

import IconGRPC from "./IconGRPC.svg";

describe("IconGRPC", () => {
  it("matches snapshot", () => {
    const aIconGRPC = shallow(<IconGRPC />).find("IconGRPC.svg");
    expect(aIconGRPC).toMatchSnapshot();
  });
});