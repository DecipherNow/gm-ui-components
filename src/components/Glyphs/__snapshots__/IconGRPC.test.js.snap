// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconGRPC matches snapshot 1`] = `<IconMock />`;
